meta-ostro/conf/distro/include/stateless.inc

```diff
@@ -498,7 +510,7 @@ passwd:         files altfiles
 group:          files altfiles
 shadow:         files altfiles
 
-hosts:          files altfiles dns
+hosts:          files mymachines resolve myhostname altfiles dns
```

Comment:
ok... have to say it was just my own reasoning for existing configuration (which might be totally wrong). So tmpfiles was copying conf file which included "files mymachines resolve myhostname" then our conf included "files altfiles dns" as next item in internal linked list of glibc. So I don't really know how these should be combined in 1 conf file? ideas are welcome...